ovos_PHAL_plugin_dotstar/__init__.py

```
@@ -143,48 +137,42 @@ def __init__(self, bus=None, config=None):
-    def main_color(self):
-        color = self.config.get(
-            "main_color", Color.from_description("Mycroft blue"))
+    def eval_color(self, color):
+        if isinstance(color, Color):
+            return color
         if isinstance(color, str):
             try:
-                color = eval(color)
-                color = Color.from_rgb(color[0], color[1], color[2])
+                color = color_from_description(color)
             except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)